proto/ots/bettor.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package ots;
+
+option go_package = "oddin.gg/ots";
+option java_package = "com.oddin.ots";
+
+import "google/protobuf/timestamp.proto";
+import "ots/commons.proto";
+
+message BettorRiskCoefficientsRequest {
+  // if recover_since is null, no updates will be recovered. If it is defined we will start to send to consumer
+  // all BettorRiskCoefficients since given timestamp value
+  optional google.protobuf.Timestamp recover_since = 1 ;
+}
+
+message BettorRiskCoefficientsResponse {
+  oneof data {
+    BettorRiskCoefficients payload = 1;
+    StreamKeepalive keepalive = 2;
+  }
+}
+
+message BettorRiskCoefficients {
+  string bettor_id = 1;
```

Comment:
I am afraid, that also ext_id must be added. + please add comments/doc what each id represents.

Reply:
OK, I added: ``id`` and ``ext_id`` with comments to be clear on what each field exactly means.